File: cucumber-java/src/test/java/io/cucumber/java/annotation/RunCucumberTest.java
package io.cucumber.java.annotation;

import org.junit.platform.suite.api.ConfigurationParameter;
import org.junit.platform.suite.api.IncludeEngines;
import org.junit.platform.suite.api.SelectPackages;
import org.junit.platform.suite.api.Suite;

import static io.cucumber.junit.platform.engine.Constants.GLUE_PROPERTY_NAME;

@Suite
@IncludeEngines("cucumber")
@SelectPackages("io.cucumber.java.annotation")
@ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "io.cucumber.java.annotation")
public class RunCucumberTest {

}

File: cucumber-kotlin-java8/src/test/kotlin/io/cucumber/kotlin/RunCucumberTest.kt
package io.cucumber.kotlin

import io.cucumber.junit.platform.engine.Constants
import org.junit.platform.suite.api.ConfigurationParameter
import org.junit.platform.suite.api.IncludeEngines
import org.junit.platform.suite.api.SelectPackages
import org.junit.platform.suite.api.Suite

@Suite
@IncludeEngines("cucumber")
@SelectPackages("io.cucumber.kotlin")
@ConfigurationParameter(key = Constants.GLUE_PROPERTY_NAME, value = "io.cucumber.kotlin")
class RunCucumberTest
